schema/workflow.yaml

```
@@ -1292,6 +1292,12 @@ $defs:
           - title: ExpressionDataSchema
             $ref: '#/$defs/runtimeExpression'
             description: An expression based event data schema.
+      data:
+        title: EventData
+        description: The event's payload data
+        anyOf:
+          - $ref: '#/$defs/runtimeExpression'
+          - type: {} 
```

I don't think this is a valid JSON schema 🤔 

I think to allow anything in JSON schema you can either use `true` or the empty object `{}`

```suggestion
          - {} 
```